BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -870,6 +870,11 @@ This document does not modify the implementation and deployment recommendations
   widespread deployment on constrained devices that lack support for
   the necessary algorithms).
 
+Although many of the recommendations provided here might also apply to QUIC insofar 
+it uses the TLS 1.3 handshake protocol, QUIC and other such secure transport protocols 
+are not the primary focus of this document. For QUIC specifically, readers are 
+referred to {{Section 9.2 of ?RFC9001}}.
```

Comment:
```suggestion
Although many of the recommendations provided here might also apply to QUIC insofar 
it uses the TLS 1.3 handshake protocol, QUIC and other such secure transport protocols 
are out of scope of this document. For QUIC specifically, readers are 
referred to {{Section 9.2 of ?RFC9001}}.
```

"Out of scope" is a lot clearer and leaves no room for too wide interpretations of the BCP's applicability.